src/strategies/uniswapv2/index.ts

```diff
@@ -0,0 +1,66 @@
+import { getAddress } from '@ethersproject/address';
+import { subgraphRequest } from '../../utils';
+
+export const author = 'stalker474';
+export const version = '0.1.0';
+
+const UNISWAPV2_SUBGRAPH_URL = {
+  1: 'https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v2'
```

Comment:
Maybe you can make others network here if Uniswap subgraph is available on testnets.